tests/contrib/scrapy/test_pipelines.py

```diff
@@ -210,6 +212,25 @@ class PipelineJSONSchemaValidator(PipelineTest):
     ]
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 4), reason="mock requires python3.4 or higher"
+)
+def test_validator_from_url():
```

Comment:
I haven't figured out how to create mock properly with the current DataTest structure so I added a simple pytest style test.